From cc5cd5a12b51bb9cbcafed5c3f693c58d333ce37 Mon Sep 17 00:00:00 2001
From: rleix <rayx.lei@intel.com>
Date: Thu, 30 Nov 2017 15:19:12 +0800
Subject: [PATCH] [WA]Remove all references about stream from OverView
 application.

Due to the Stream and Overview are deprecated, but currently
Google only remove the Stream and the Overview still exist.
In order to fix the Overview crash when sending the intent call
to Stream. Remove the intent call to Stream.

Change-Id: I2394e898faa753bfb71fbdcf2192fe4739ee0804
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-53321
Signed-off-by: Lei,RayX <rayx.lei@intel.com>
---
 src/com/android/car/overview/StreamOverviewActivity.java | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/com/android/car/overview/StreamOverviewActivity.java b/src/com/android/car/overview/StreamOverviewActivity.java
index 3f1e191..8a868dd 100644
--- a/src/com/android/car/overview/StreamOverviewActivity.java
+++ b/src/com/android/car/overview/StreamOverviewActivity.java
@@ -122,13 +122,13 @@ public class StreamOverviewActivity extends Activity {
 
     private void startPermissionsActivity(boolean checkPermissionsOnly) {
         // Start StreamService's permission activity before binding to it.
-        Intent intent = new Intent();
+        /*Intent intent = new Intent();
         intent.setComponent(new ComponentName(
                 getString(R.string.car_stream_item_manager_package_name),
                 getString(R.string.car_stream_item_manager_permissions_activity)));
         intent.putExtra(StreamConstants.STREAM_PERMISSION_CHECK_PERMISSIONS_ONLY,
                 checkPermissionsOnly);
-        startActivityForResult(intent, PERMISSION_ACTIVITY_REQUEST_CODE);
+        startActivityForResult(intent, PERMISSION_ACTIVITY_REQUEST_CODE);*/
     }
 
     @Override
@@ -141,7 +141,7 @@ public class StreamOverviewActivity extends Activity {
         }
     }
 
-    @Override
+    /*@Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == PERMISSION_ACTIVITY_REQUEST_CODE) {
             // onResume is called after onActivityResult, if the permissions activity has
@@ -156,7 +156,7 @@ public class StreamOverviewActivity extends Activity {
                 mPageListView.setVisibility(View.GONE);
             }
         }
-    }
+    }*/
 
     @Override
     protected void onDestroy() {
-- 
1.9.1

